native/packages/react-native-bpk-component-card/src/BpkCard-test.common.js

```diff
@@ -15,7 +15,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
Why is this newline removed?

Reply:
ah! I've removed the eslint disable and I forgot to put back the new line 😅